SpatialGDK/Source/SpatialGDKEditor/Private/SpatialGDKEditorCloudLauncherSettings.cpp

```diff
@@ -0,0 +1,160 @@
+// Copyright (c) Improbable Worlds Ltd, All Rights Reserved
+#include "SpatialGDKEditorCloudLauncherSettings.h"
+
+#include "Dom/JsonObject.h"
+#include "Internationalization/Regex.h"
+#include "Misc/FileHelper.h"
+#include "Serialization/JsonReader.h"
+#include "Serialization/JsonSerializer.h"
+#include "Settings/LevelEditorPlaySettings.h"
+#include "SpatialGDKEditorSettings.h";
+#include "Templates/SharedPointer.h"
+
+USpatialGDKEditorCloudLauncherSettings::USpatialGDKEditorCloudLauncherSettings(const FObjectInitializer& ObjectInitializer)
+	: Super(ObjectInitializer)
+{
+	const USpatialGDKEditorSettings * SpatialEditorSettings = GetDefault<USpatialGDKEditorSettings>();
+	SnapshotPath.FilePath = FPaths::Combine(SpatialEditorSettings->GetSpatialOSSnapshotFolderPath(), SpatialEditorSettings->GetSpatialOSSnapshotFile());
+	ProjectName = GetProjectNameFromSpatial();
+}
+
+FString USpatialGDKEditorCloudLauncherSettings::ToString()
```

Comment:
This function shouldn't be needed. I removed it for the GDK editor settings:)

Reply:
Awesome - removed it 😃